lib/sshkit/scp/command.ex

```diff
@@ -3,22 +3,29 @@ defmodule SSHKit.SCP.Command do
 
   import SSHKit.Utils
 
-  def build(mode, path, options) do
-    scp = case mode do
-      :upload -> 'scp -t'
-      :download -> 'scp -f'
-    end
+  @flags [verbose: '-v', preserve: '-p', recursive: '-r']
 
-    flags = [verbose: '-v', preserve: '-p', recursive: '-r']
+  def build(:upload, path, options) do
+    scp('-t', path, options)
```

Comment:
are there long versions for this flag? they might be more documenting, so that the occasional reader `*`cough`*` me `*`cough`*` gets what this is doing :)